src/js/components/AppListItemComponent.jsx

```diff
@@ -105,8 +105,8 @@ var AppListItemComponent = React.createClass({
           {parseFloat(model.totalCpus).toFixed(1)}
         </td>
         <td className="text-right total ram">
-          <span title={`${model.totalMem}MB`}>
-            {`${ViewHelper.convertMegabytesToString(model.totalMem)}`}
+          <span title={`${model.totalMem} MiB`}>
+            {`${Util.filesize(model.totalMem * Math.pow(1024, 2), 0)}`}
```

Comment:
How can I know that I need to do this operation to be able to call `Util.filesize`? Imho the function should perform such transformation, or the function name should be more expressive. `filesize` is too generic to express the need for a specific unit when passing the size.